lib/std/zip.zig

```diff
@@ -1,8 +1,8 @@
-/// The .ZIP File Format Specification is found here:
-///    https://pkwaredownloads.blob.core.windows.net/pem/APPNOTE.txt
-///
-/// Note that this file uses the abbreviation "cd" for "central directory"
-///
+//! The .ZIP File Format Specification is found here:
+//!    https://pkwaredownloads.blob.core.windows.net/pem/APPNOTE.txt
+
+// Note that this file uses the abbreviation "cd" for "central directory"
```

Comment:
```suggestion
//!
//! Note that this file uses the abbreviation "cd" for "central directory"
```

I think this applies to some fields that are user-facing, so it seems reasonable to make this part of the doc comment.